src/applause-button.js

```diff
@@ -1,33 +1,27 @@
-import "document-register-element/build/document-register-element";
+import "@ungap/custom-elements";
 
 const VERSION = "3.3.0";
 const API = "https://api.applause-button.com";
 
 const getClaps = (api, url) =>
-  // TODO: polyfill for IE (not edge)
-  fetch(`${api}/get-claps` + (url ? `?url=${url}` : ""), {
-    headers: {
```

Comment:
I assume we no longer need the Content-Type header for the get request?

Reply:
Nope, cos we're not sending any content. Probably a copy-paste job from the POST request 😊